test/test-forms.json

```diff
@@ -2,5 +2,9 @@
     {
         "name": "Outlook",
         "url": "https://outlook.live.com/owa/?authRedirect=true&nlp=1"
+    },
+    {
+        "name": "GitHub",
+        "url": "https://github.com/login"
```

Comment:
So, as far as I know, this doesn't do anything yet 😄 But I added it here so that we have it on the list if/when we start testing actual sites in the tests.

Reply:
#9 😁